drivers/regulator/regulator_nxp_vref.c

```diff
@@ -165,7 +164,7 @@ static int regulator_nxp_vref_get_voltage(const struct device *dev,
 	/* Linear range index is the register value */
 	idx = (base->UTRIM & VREF_UTRIM_TRIM2V1_MASK) >> VREF_UTRIM_TRIM2V1_SHIFT;
 
-	ret = linear_range_get_value(&utrim_range, base->UTRIM, volt_uv);
+	ret = linear_range_get_value(&utrim_range, idx, volt_uv);
```

Comment:
This looks like a separate fix from the `gnd_sel` issue- maybe this line needs to be in another commit? This will make it easier for users in the future to follow *why* this change was made

Reply:
yes, this is bug fix, I can add one commit to do this